src/GenesisSeek.cpp

```diff
@@ -20,6 +21,9 @@ SeekControl::SeekControl(BRect rect, const char *name, BMessage* msg)
 	GetFont(&font);
 	SetDivider(font.StringWidth("Seek for")+4);
 
+	font.SetSize(10);
```

Comment:
Hardcoding font size is not a good idea, since our HiDPI "support" relies on global font size settings.

Query for default font size, then scale it.

Reply:
I know, the thing is that the seek panel have limited space. When activated, it is displayed over "info panel" in the main window. And main window do not use layout kit....

The ideal solution to rewrite everything with layout kit. I thought that, as issue #33 have been already opened, this solution is better than nothing. I will try to rewrite everything to layout kit with scaling in mind - but that is task for another day...

Reply:
OK.